### DISCUSSION TITLE
When I print the global object in the getServerSideProps function, I see some attributes that nodejs does not have, such as fetch, Headers, Request, Response, why are there these, can I use it safely?

### DISCUSSION THREAD

**Discussion by lvzegeng** [vercel/next.js#13830] [Help] [answered]
When I print the global object in the getServerSideProps function, I see some attributes that nodejs does not have, such as fetch, Headers, Request, Response, why are there these, can I use it safely?

当我在getServerSideProps函数打印global对象时，我看到了一些nodejs没有的属性，如fetch、Headers、Request、Response，为什么会有这些呢，我能安全的使用吗？

**Comment by wereHamster** [comment, marked as answer]
https://nextjs.org/blog/next-9-4#improved-built-in-fetch-support
